modules/setting/setting.go

```diff
@@ -136,10 +136,11 @@ var (
 	}
 
 	LFS struct {
-		StartServer     bool   `ini:"LFS_START_SERVER"`
-		ContentPath     string `ini:"LFS_CONTENT_PATH"`
-		JWTSecretBase64 string `ini:"LFS_JWT_SECRET"`
-		JWTSecretBytes  []byte `ini:"-"`
+		StartServer           bool   `ini:"LFS_START_SERVER"`
+		ContentPath           string `ini:"LFS_CONTENT_PATH"`
+		JWTSecretBase64       string `ini:"LFS_JWT_SECRET"`
+		JWTSecretBytes        []byte `ini:"-"`
+		HTTPAuthExpiryMinutes int    `ini:"LFS_HTTP_AUTH_EXPIRY_MINUTES"`
```

Comment:
I find more logic and allow more flexible configuration to set this directly as time.Duration you can use mirror option like MIN_INTERVAL as an example and use a more short option name LFS_HTTP_AUTH_EXPIRY.